README.md

```diff
@@ -1 +1,18 @@
-# Frends.SFTP
\ No newline at end of file
+# Frends.SFTP
+
+Frends tasks for tranfering files through SFTP connection.
```

Comment:
This line could be something like Frends tasks for SFTP operations, since the package consists tasks which will do more than just transfer files (like list files).

Reply:
Fixed